Added LSM6DSV to imu-comparison.md (#277)
SlimeVR/SlimeVR-Docs-Site: src/diy/imu-comparison.md

> Please note, if you are looking for BNO085s SlimeVR may have [spares for sale that can be found here](https://shop.slimevr.dev/products/slimevr-imu-module-bno085).

---
## LSM6DSV
The LSM6DSV is a new and <b>experimental</b> IMU for DIY SlimeVR.

> Offers very good reset times even in bad magnetic enviroments and for fast movements, dancing. Low availability because the chip is new. 

Reviewer: ```suggestion
Can offer very good reset times even in bad magnetic enviroments and for fast movements, dancing. Low availability because the chip is new. 
```

|Reset time |Cost |Availability|Build quality|
|:---------:|:---:|:----------:|:-----------:|

> |60 min     |~$15 |Insufficient|Excellent    |

Reviewer: I have some issues with this.

- Reset times are only up to 60 minutes in the best case scenario, for a lot of users it's significantly worse (and depends heavily on calibration), so I'd suggest putting down a range instead (like 45-60 but even that's optimistic)
- Cost doesn't make much sense, seeing as there's no single go-to source for modules at this time. I'd suggest changing it to be the price of the raw chip from suppliers like Mouser, Digikey, etc.
- Build quality also cannot be judged for the same reasons as cost.

Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-half-o"></i>

> |Pros          |Cons                                   |
> |--------------|---------------------------------------|
> |Accurate               |Low availability              |
> |Reliable               |Experimental Beta Firmware    |
> |Consistent Reset times |                              |
> |Single time calibration|                              |

Reviewer: ```suggestion
|Pros                   |Cons                                                          |
|-----------------------|--------------------------------------------------------------|
|Accurate               |[Requires manual calibration the first time](#imu-calibration)|
|Reliable               |Low availability                                              |
|Single time calibration|Experimental Beta Firmware                                    |
|                       |Performance data not taken from survey                        |
```

I wouldn't keep the "consistent reset times" line since performance is so heavily dependent on calibration.

>  

Reviewer: ```suggestion

 > Please note, that the main branch of SlimeVR firmware does not support this IMU yet, and running a custom fork is required.
 
```
